### DISCUSSION TITLE
how do i get around?

### DISCUSSION THREAD

**Discussion by wonuola-y** [community/community#17907] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I need help understanding how to get around here

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to the Community 👋🏼 These are discussions where you can share suggestions for how GitHub products should be improved and discuss those improvements with the community, including members of the GitHub product team. Check out [Making suggestions](https://github.com/github-community/community#making-suggestions) to learn how to provide feedback.
